### PR TITLE
New version: WHONET.WHONET version 24.9.25

### PR DESCRIPTION
**Pull request by AClark-WHONET**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/175160)

### CODE REVIEW

File: manifests/w/WHONET/WHONET/24.9.25/WHONET.WHONET.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- BacLink
+- Bacteriology
+- CLSI
+- EUCAST
+- GLASS
+- Laboratory
+- Microbiology
+- Resistance
+- WHO
+ReleaseNotes: Corrects a problem handling breakpoints for serovar groups: ECO, HIN, NME.
```
Comment (stephengillie) [inline manifests/w/WHONET/WHONET/24.9.25/WHONET.WHONET.locale.en-US.yaml:34]:
```suggestion
ReleaseNotes: Corrects a problem handling breakpoints for serovar groups\: ECO, HIN, NME.
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
HI @AClark-WHONET,

Automatic Validation ended with:

> ##[error] (Line: 34, Col: 73, Idx: 1900) - (Line: 34, Col: 73, Idx: 1900): While scanning a plain scalar value, found invalid mapping.

I'm not a YAML expert, but I believe the colon might need to be escaped.

**Comment by AClark-WHONET** [comment]
> HI @AClark-WHONET,
> 
> Automatic Validation ended with:
> 
> > ##[error] (Line: 34, Col: 73, Idx: 1900) - (Line: 34, Col: 73, Idx: 1900): While scanning a plain scalar value, found invalid mapping.
> 
> I'm not a YAML expert, but I believe the colon might need to be escaped.

Thanks Stephen.

File: manifests/w/WHONET/WHONET/24.9.25/WHONET.WHONET.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- BacLink
+- Bacteriology
+- CLSI
+- EUCAST
+- GLASS
+- Laboratory
+- Microbiology
+- Resistance
+- WHO
+ReleaseNotes: Corrects a problem handling breakpoints for serovar groups\: ECO, HIN, NME.
```
Comment (stephengillie) [inline manifests/w/WHONET/WHONET/24.9.25/WHONET.WHONET.locale.en-US.yaml:34]:
```suggestion
ReleaseNotes: "Corrects a problem handling breakpoints for serovar groups: ECO, HIN, NME."
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Apparently I was wrong about escaping, and an Internet search brings several recommendations to quote the string contents.

File: manifests/w/WHONET/WHONET/24.9.25/WHONET.WHONET.locale.en-US.yaml (hunk 2 above)
Comment (AClark-WHONET) [reply manifests/w/WHONET/WHONET/24.9.25/WHONET.WHONET.locale.en-US.yaml]: Thanks again!

**Review by stephengillie — APPROVED** [review]

**Pull request merged**